spark/build.gradle.kts

```diff
@@ -75,6 +77,19 @@ dependencyGuard {
     configuration("releaseRuntimeClasspath")
 }
 
+tasks.dokkaHtml.configure {
+    moduleName.set("Spark")
+    outputDirectory.set(rootProject.buildDir.resolve("dokka"))
+    dokkaSourceSets.configureEach {
+        // List of files or directories containing sample code (referenced with @sample tags)
+        // samples.from("samples/basic.kt", "samples/advanced.kt")
```

Comment:
@soulcramer here is the setting to configure Dokka samples 😄

Reply:
👀 
It'll be nice when we'll be able to use it with the sample app ☺️